app/models/category.rb

```diff
@@ -0,0 +1,4 @@
+class Category < ApplicationRecord
+  has_many :motorcycles
+  validates :categname, presence: true, uniqueness: true
+end
```

Comment:
I would suggest that the description 'category' be spelled out to avoid any confusion instead of 'categname'

Reply:
Ok, I will change it